### PR TITLE
Add support removal of Firefox-only 'version' parameter in 'type' attribute of <script>

### PR DESCRIPTION
**Pull request by chrisdavidmills**
…script element type attribute

### CODE REVIEW

**Comment by teoli2003** [comment]
@chrisdavidmills This looks strange… Do you have the bug #?

**Comment by chrisdavidmills** [comment]
Yup: https://bugzilla.mozilla.org/show_bug.cgi?id=1428745

File: html/elements/script.json [hunk 1]
```diff
@@ -549,6 +549,56 @@
                 "deprecated": false
               }
             }
+          },
+          "version_parameter": {
+            "__compat": {
+              "description": "The <code>version</code> parameter of the <code>type</code> attribute.",
```
Comment (teoli2003) [inline html/elements/script.json]: Thanks for the bug #, it makes sense now.

No dot at the end of description.

Comment (chrisdavidmills) [reply html/elements/script.json]: Done!

**Pull request merged**